aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -79,7 +79,7 @@ c10::optional<OperatorHandle> Dispatcher::findOp(const OperatorName& overload_na
 void Dispatcher::waitForDef(const FunctionSchema& schema) {
   using namespace std::chrono_literals;
   std::unique_lock<std::mutex> lock(mutex_);
-  bool r = cond_var_.wait_for(lock, 2s, [&]{
```

Comment:
oh, what is this?

Reply:
This is the setup that enables multiple interpreters to handle op registration.
It requires waiting for ops to pop up with a given timeout.
I'll remove this change if my latest hack works. 🤞